src/PKSim.Core/Model/AggregationFunctions.cs

```diff
@@ -28,40 +28,43 @@ public static class AggregationFunctions
 
```

Comment:
I think this is because of the range that is being created (from 5% to 95% => 90) as opposed to (2.5% => 97.5%). I did not choose that nomenclature, it was part of the specs for 5.3...